ddp-workspace/projects/toolkit-prion/src/lib/components/footer/prionFooter.component.ts

```diff
@@ -44,8 +44,9 @@ export class PrionFooterComponent extends FooterComponent {
     private _windowRef: WindowRef,
     @Inject('toolkit.toolkitConfig') private _toolkitConfiguration: ToolkitConfigurationService,
     private _auth0Adapter: Auth0AdapterService,
-    private _session: SessionMementoService) {
-    super(_dialog, _analytics, _windowRef, _toolkitConfiguration);
+    private _session: SessionMementoService,
+    private translate: NGXTranslateService) {
```

Comment:
Is prefixing with an underscore the convention for things that's not used in current class but need to be injected to be passed to parent class?

Reply:
For what I have seen, Charlotte has adopted that convention for private instance variables.